starfyre/parser.py

```diff
@@ -117,16 +117,40 @@ def handle_starttag(self, tag, attrs):
         if tag not in self.generic_tags and tag not in self.components:
             raise UnknownTagError(f'Unknown tag: "{tag}". Please review line {self.lineno} in your "{self.component_name}" component in the pyml code.')
 
-        component = Component(
-            tag,
-            props,
-            [],
-            event_listeners,
-            state,
-            js=self.js,
-            css=self.css,
-            uuid=uuid4(),
-        )
+        # component = Component(
+        #     tag,
+        #     props,
+        #     [],
+        #     event_listeners,
+        #     state,
+        #     js=self.js,
+        #     css=self.css,
+        #     uuid=uuid4(),
+        # )
```

Comment:
Hey @SuelenKarbivnychyy 👋 

One nit - These comments are extra 😅

Reply:
> Hey @SuelenKarbivnychyy 👋
> 
> One nit - These comments are extra 😅

Sorry, I forgot to delete it after test 😅